assemblyline/odm/models/alert.py

```diff
@@ -165,41 +173,45 @@ def __hash__(self) -> int:
-    label = odm.List(odm.Keyword(), copyto="__text__", default=[], description="List of labels applied to the alert")
-    metadata = odm.FlattenedObject(default={}, store=False, description="Metadata submitted with the file")
-    owner = odm.Optional(odm.Keyword(), description="Owner of the alert")
-    priority = odm.Optional(odm.Enum(values=PRIORITIES), description="Priority applied to the alert")
-    reporting_ts = odm.Date(description="Alert creation timestamp")
-    submission_relations = odm.sequence(odm.compound(Relationship), description="Describes relationships "
-                                        "between submissions used to build this alert")
-    sid = odm.UUID(description="Submission ID related to this alert")
-    status = odm.Optional(odm.Enum(values=STATUSES), description="Status applied to the alert")
-    ts = odm.Date(description="File submission timestamp")
-    type = odm.Keyword(description="Type of alert")
-    verdict = odm.Compound(Verdict, default={}, description="Verdict Block")
-    events = odm.sequence(odm.compound(Event), default=[], description="An audit of events applied to alert")
-    workflows_completed = odm.Boolean(default=False, description="Have all workflows ran on this alert?")
+    alert_id = odm.Keyword(copyto="__text__", description="Unique identifier for the alert.")
+    al = odm.compound(ALResults, description="Contains the results of the Assemblyline analysis for the alert.")
+    archive_ts = odm.Optional(odm.Date(description="Timestamp indicating when the alert was archived in the system."))
+    attack = odm.Compound(Attack, description="Structured data representing MITRE ATT&CK information associated with the alert.")
+    classification = odm.Classification(description="Security classification level of the alert.")
+    expiry_ts = odm.Optional(odm.Date(store=False), description="Timestamp indicating when the alert is scheduled to expire from the system.")
+    extended_scan = odm.Enum(values=EXTENDED_SCAN_VALUES, description="Indicates the status of an extended scan, if applicable. Extended scans are additional analyses performed after the initial analysis.")
+    file = odm.Compound(File, description="Information about the file associated with the alert.")
+    filtered = odm.Boolean(default=False, description="Indicates whether the alert results are filtered out based on system or user configurations.")
+    heuristic = odm.Compound(Heuristic, description="Data regarding the heuristics that triggered the alert.")
+    label = odm.List(odm.Keyword(), copyto="__text__", default=[], description="Labels assigned to the alert for categorization and filtering.")
+    metadata = odm.FlattenedObject(default={}, store=False, description="Additional metadata provided with the file at the time of submission.")
+    owner = odm.Optional(odm.Keyword(), description="The username or system component that owns or is responsible for the alert.")
```

Comment:
Does `owner` only represent a human, or it cna represent a system?

Reply:
A human can choose to take ownership over an alert, but I think by default it belongs to the system

Reply:
Potential new description:
```Specifies the user or system component that has taken ownership of the alert. If no user has claimed the alert, it remains under system ownership with no specific user associated, indicated by a value of `None`.```